code/modules/clothing/shoes/_shoes.dm

```diff
@@ -178,4 +183,34 @@
 		S.alpha = 127 * shine / 100
 		S.blend_mode = BLEND_ADD
 		overlay.overlays += S
-	. = ..()
\ No newline at end of file
+	. = ..()
+
+// Interactions
```

Comment:
I can't completely avoid messing with the cuffs a bit in this PR though. Because, handcuffs use the health var to tell if someone has broken out of the cuffs or not. But, the sec handcuffs can't be broken out of, and yet should be able to take damage.. So this is kind of cursed.

Reply:
Alright. I removed the changes in the shoes file.